cmd/kpod/exec.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	execFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "detach, d",
+			Usage: "detached mode: command will run in the background",
+		},
+		cli.StringSliceFlag{
+			Name:  "env, e",
+			Usage: "set environment variables",
+		},
+		cli.BoolFlag{
+			Name:  "tty, t",
+			Usage: "Allocate a pseudo TTY",
+		},
+		cli.StringFlag{
+			Name:  "user, u",
+			Usage: "Sets the username (UID) and groupname (GID) for the specified command",
+		},
+	}
+	execDescription = `
+   kpod exec
+
+   Run a command in a running container.  The command will only run while the primary
+	PID in the container is running.
+`
+
+	execCommand = cli.Command{
+		Name:        "exec",
+		Usage:       "Run a command in a running container",
+		Description: execDescription,
+		Flags:       execFlags,
+		Action:      execCmd,
+		ArgsUsage:   "[OPTIONS} CONTAINER-NAME COMMAND",
+	}
+)
+
+func execCmd(c *cli.Context) error {
+	args := c.Args()
+	detachFlag := c.Bool("detach")
+	envFlags := c.StringSlice("env")
```

Comment:
No need for variable, it is only used once.

Reply:
fixed